README.md

```diff
@@ -76,6 +77,35 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `chectl devfile:generate`
+
+generate and print a devfile to stdout given some Kubernetes resources and other Che workspaces features (project, language-support, commands etc...)
+
+```
+USAGE
+  $ chectl devfile:generate
+
+OPTIONS
+  -h, --help                 show CLI help
+  --command=command          Command to include in the workspace
+  --dockerImage=dockerImage  dockerImage component to include to the workspace
```

Comment:
Maybe it would be better to use `--dockerimage` since it's a value of type for dockerimage component?